app/code/Magento/ConfigurableProduct/Pricing/Render/FinalPriceBox.php

```diff
@@ -77,4 +94,12 @@ public function hasSpecialPrice()
         }
         return false;
     }
+
+    /**
+     * @inheritdoc
+     */
+    protected function isApplySalableCheck(SaleableInterface $salableItem): bool
```

Comment:
The new protected method is a backward incompatible change, that cannot be delivered to patch release. What do you think about overriding the `_toHtml` method instead?

Reply:
To be honest I do not remember why I have introduce this method :)
Will take a look on it on the next week (I’m on vacation now)
Thank you!